### PR TITLE
Some reformatting and cleanup.

### PR DESCRIPTION
**Pull request by Soheab**
- Added `B?` (dev) and `B!` for mobile users.
- `b!delete` can now accept a bot mention or id.
- Removed changing status from on_ready, made a ext.tasks for it instead.
- Changed all `is None` , `is not None`, `!= []` and `== None` to `not ...` and `is ...` .
- Changes to staff command: made everything into a dict with lists and using get_member.
- Stopped shadowing built-in names like `id`.
- Combined some lines to less lines or something like that.
- Removed separate cog for the error handler, moved to events.py.
- Changes to rulesninfo command: made server and bot rules in a list and using `enumerate()` to "add" it to the embed.
- Added space between every `=` (pycharm).
- Made use of `discord.utils.oauth_url()` for invites.

Every change is tested, the dicts and lists, union, ext.tasks and enumerate() things.

### CODE REVIEW

File: cogs/checks.py [hunk 1]
```diff
@@ -16,4 +17,5 @@ async def predicate(ctx):
             return False
         else:
             return True
-    return commands.check(predicate)
\ No newline at end of file
+
+    return commands.check(predicate)
```
Comment (GavynStanley) [inline cogs/checks.py:21]: Why such a big space?

File: cogs/admin.py [hunk 2]
```diff
@@ -128,136 +137,159 @@ async def update(self, ctx):
...
+        await links.edit(embed = links_embed)
+        await faqs.edit(embed = faq_embed)
+        await ctx.send(f"Updated all embeds in {channel.mention}")
 
     @checks.main_guild_only()
-    @commands.has_permissions(administrator=True)
-    @commands.command()
-    async def votesreset(self, ctx, *, message=None):
+    @commands.has_permissions(administrator = True)
+    @commands.command(name = "votesreset")
```
Comment (GavynStanley) [inline cogs/admin.py]: I don't like the name of this

**Review by GavynStanley — CHANGES_REQUESTED** [review]
Imports were sorted in a certain way as well, please revert import sorting.

**Comment by Soheab** [comment]
> Imports were sorted in a certain way as well, please revert import sorting.

done

**Review by GavynStanley — APPROVED** [review]

**Comment by GavynStanley** [comment]
Awesome

**Pull request merged**